script.js

```diff
@@ -1,35 +1,42 @@
-const sideNav = document.getElementById("sideNav")
+/* Toggle SideNav Logic */
+const sideNav = document.getElementById('sideNav');
 
 function openNav(open) {
-  sideNav.style.width = "100%";
+  document.getElementById('sideNav').style.width = '100%';
+  sideNav.style.width = '100%';
   return open;
 }
 
+document.getElementById('toggleSpan').onclick(openNav);
+
 function closeNav(close) {
-  sideNav.style.width = "0";
+  document.getElementById('sideNav').style.width = '0';
+  sideNav.style.width = '0';
   return close;
 }
+document.getElementById('xOut').onclick(closeNav);
 
-// CONTACT VALIDATION
-
-const form = document.querySelector("form");
-form.addEventListener("submit", handleSubmit);
-
+// Contact for validation and giving error feedback
+const form = document.querySelector('form');
 function handleSubmit(event) {
   event.preventDefault();
 
-  const emailInput = document.querySelector("#email");
+  const emailInput = document.querySelector('#email');
   const email = emailInput.value;
 
   if (email === email.toLowerCase()) {
-    // If iy fits requirement[lowercase], ahead
+    // If it fits requirement[lowercase], ahead
     form.submit();
   } else {
     // Otherwise show dynamic error below
-    const errorMessage = document.createElement("p");
-    errorMessage.classList.add("errorMsg");
-    errorMessage.textContent =
-      "Opps! An error has occured, your email must be lowercase.";
+    let errorMessage = form.querySelector('.errorMsg');
+    if (errorMessage) {
+      form.removeChild(errorMessage);
+    }
+    errorMessage = document.createElement('p');
+    errorMessage.classList.add('errorMsg');
```

Comment:
- [ ] [OPTIONAL] You have done great to ensure the display of the error message. However, if you notice most of the errors in websites are styled with reddish color. Kindly consider using one of the reddish colors ☺️.